systemd.yaml

```diff
@@ -369,11 +369,14 @@ subpackages:
         - ${{package.name}}
         - tzdata
         - mount
+        - kbd
+        - agetty
     pipeline:
       - runs: |
           mkdir -p ${{targets.subpkgdir}}/etc ${{targets.subpkgdir}}/sbin
-          ln -s ../usr/lib/systemd/systemd ${{targets.subpkgdir}}/sbin/init
-          ln -sf ../usr/share/zoneinfo/UTC ${{targets.subpkgdir}}/etc/localtime
+          ln -sf ../usr/lib/systemd/systemd ${{targets.subpkgdir}}/sbin/init
+          ln -sf ../run/systemd/resolve/stub-resolv.conf "${{targets.subpkgdir}}/etc/resolv.conf"
```

Comment:
just a comment. '-s' rather than '-sf' is nicer in that it will fail if something else changes and this is already there.  ie, if something above moved this around or something produced content there, then the build would fail rather than finding out later that we masked some content unintentionally.